src/instructlab/model/backends/vllm.py

```diff
@@ -259,6 +282,20 @@ def build_vllm_cmd(
             verify_template_exists(path)
             vllm_cmd.extend(["--chat-template", chat_template])
 
+    # Auto-detect whether model is quantized w/ bitsandbytes and add potentially missing args
+    quant_arg_present = contains_argument("--quantization", vllm_args)
+    load_arg_present = contains_argument("--load-format", vllm_args)
+    eager_arg_present = contains_argument("--enforce-eager", vllm_args)
+    if not (quant_arg_present and load_arg_present and eager_arg_present):
+        if is_bnb_quantized(model_path):
+            if not quant_arg_present:
+                vllm_cmd.extend(["--quantization", "bitsandbytes"])
+            if not load_arg_present:
+                vllm_cmd.extend(["--load-format", "bitsandbytes"])
+            # Currently needed to retain generation quality w/ 4-bit bnb + vLLM (bypass graph bug)
```

Comment:
Is there a bug to refer to in the comment? Please add where one can find more details.